using_images/other_images/jenkins.adoc

```diff
@@ -417,3 +417,25 @@ Jenkins job] that is pre-configured in the Jenkins image utilizes the
 plug-in for creating CI/CD flows for {product-title} in Jenkins.
 
 See the https://github.com/openshift/jenkins-plugin/[the plug-in's README] for a detailed description of what is available.
+
+== OpenShift Client Plug-in
+
+The experiences gained working with users of the OpenShift Pipeline plug-in, coupled with the rapid evolution of both Jenkins
+and OpenShift, have provided valuable insight into how to best provide {product-title} integration from Jenkins jobs.
```

Comment:
hmm ... not seeing with my editor ... I'll switch to a couple of others, see if it emerges